tests/ci/cdk/cdk/github_codebuild_stack.py

```diff
@@ -52,13 +52,21 @@ def __init__(self,
                             iam.ManagedPolicy.from_aws_managed_policy_name("AmazonEC2ContainerRegistryReadOnly")
                         ])
 
+        # Define timeout.
```

Comment:
It looks like the second check is dependent on some property from the CodeBuild definition, but is there any way to combine this check with the second check that also sees if `env_type is ARM`?

Reply:
No, because 
*`timeout` is a supported property of CDK CodeBuild project so it can be passed when constructing CDK project obj.
* the 2nd check is to provide a **workaround** (override not exposed property). The override can only happen after CDK project obj construction.

Reply:
I did a research but still cannot find a way to get rid of the workaround.